fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -899,12 +904,14 @@ struct EncryptedRangeFileWriter : public IRangeFileWriter {
 		if (k.size() > 0 &&
 		    (!self->cipherKeys.headerCipherKey.present() || !self->cipherKeys.headerCipherKey.get().isValid() ||
 		     !self->cipherKeys.textCipherKey.isValid())) {
-			wait(updateEncryptionKeysCtx(self, k));
+			wait(updateEncryptionKeysCtx(self, k, true));
```

Comment:
nit: better to use FDB_XX_BOOLEAN_PARAM than passing `true` or `false`. Feel free to do this as a follow up PR.